client/src/locales/translations/es_ES.json

```diff
@@ -224,7 +224,7 @@
         "linkedin": "URL de LinkedIn",
         "website": "Página web"
       },
-      "emailMaxLength": "La dirección de correo electrónico no debe exceder los 200 caracteres",
+      "emailMaxLength": "La dirección de correo electrónico no debe exceder los 254 caracteres",
```

Comment:
Just a note, there's no need to touch these other translation files manually. These changes won't be captured in Lokalise and the next time these languages are exported, the previous translation (200 characters) will show up again. 

Only the changes in `en_US` will get picked up by Lokalise and from there, translators will have to change the message of all the other languages. And then we can export the changes. That's the proper way to do it and there will be a lag / desync between the languages. But technically we can make the changes here and on Lokalise so there isn't a desync. 

So keeping these changes are fine, just wanted to give you a heads up!

We should have implemented this differently so you can inject the number of characters into the translation.